third_party/txt/BUILD.gn

```diff
@@ -185,6 +186,77 @@ source_set("txt_test_utils") {
   ]
 }
 
+test_fixtures("txt_fixtures") {
+  fixtures = [
+    "third_party/fonts/Bold.ttf",
+    "third_party/fonts/Bold.ttx",
+    "third_party/fonts/BoldItalic.ttf",
+    "third_party/fonts/BoldItalic.ttx",
+    "third_party/fonts/ColorEmojiFont.ttf",
+    "third_party/fonts/ColorEmojiFont.ttx",
+    "third_party/fonts/ColorTextMixedEmojiFont.ttf",
+    "third_party/fonts/ColorTextMixedEmojiFont.ttx",
+    "third_party/fonts/DroidSerif.ttf",
+    "third_party/fonts/Emoji.ttf",
+    "third_party/fonts/Emoji.ttx",
+    "third_party/fonts/HomemadeApple-LICENSE.txt",
+    "third_party/fonts/HomemadeApple.ttf",
+    "third_party/fonts/Italic.ttf",
+    "third_party/fonts/Italic.ttx",
+    "third_party/fonts/Ja.ttf",
+    "third_party/fonts/Ja.ttx",
+    "third_party/fonts/Katibeh-LICENSE.txt",
+    "third_party/fonts/Katibeh-Regular.ttf",
+    "third_party/fonts/Ko.ttf",
+    "third_party/fonts/Ko.ttx",
+    "third_party/fonts/MultiAxis.ttf",
+    "third_party/fonts/MultiAxis.ttx",
+    "third_party/fonts/NoCmapFormat14.ttf",
+    "third_party/fonts/NoCmapFormat14.ttx",
+    "third_party/fonts/NoGlyphFont.ttf",
+    "third_party/fonts/NoGlyphFont.ttx",
+    "third_party/fonts/NotoColorEmoji-LICENSE.txt",
+    "third_party/fonts/NotoColorEmoji.ttf",
+    "third_party/fonts/NotoNaskhArabic-LICENSE.txt",
```

Comment:
We don't actually depend on these LICENSE files, so they don't need to be here unless you want them here just to verify their existence.

Reply:
Patched.